src/compose/ComposeText.js

```diff
@@ -71,6 +82,30 @@ class ComposeText extends React.Component {
     });
   }
 
+  showErrorAlert = (message, title) => {
+    Alert.alert(
+         title,
+         message,
+         [{ text: 'OK', onPress: () => {} }],
+         { 'cancelable': true } // eslint-disable-line spellcheck/spell-checker
+       );
+  }
+
+  handleEdit = async () => {
+    const { auth, editMessage, cancelEditMessage } = this.props;
+    if (editMessage.content.raw_content === this.state.text) {
```

Comment:
Just do a return.
I don't think there is a single person that cares for the error.
This is too much of a 'developer' thinking ;)